website/docs/r/vpc_subnet_v1.html.md

```diff
@@ -25,6 +25,16 @@ resource "huaweicloud_vpc_subnet_v1" "subnet_v1" {
   gateway_ip = "${var.subnet_gateway_ip}"
   vpc_id = "${huaweicloud_vpc_v1.vpc_v1.id}"
 }
+
+resource "huaweicloud_vpc_subnet_v1" "subnet_with_tags" {
```

Comment:
Indent two spaces for each nesting level.
please see: https://www.terraform.io/docs/configuration/style.html